src/components/ProductPopover.vue

```diff
@@ -31,6 +32,9 @@ export default defineComponent({
     }),
   },
   methods: {
+    onlyRevert() {
+      this.isVirtual ? this.onlyRevertSingleProduct() : this.onlyRevertParentProduct();
```

Comment:
If isVirtual means it is virtual product, I think we should use onlyRevertParentProduct and for variant we should use onlyRevertSingleProduct

Reply:
Updated the code.